test/unit-tests/yok.ts

```diff
@@ -601,9 +601,12 @@ describe("yok", () => {
 	});
 
 	function deleteDirectory(directory: string): boolean {
-		shelljs.rm("-rf", directory);
-		let err = shelljs.error();
-		return err === null;
+		try {
+			shelljs.rm("-rf", directory);
```

Comment:
OK. This is valid for NativeScript CLI, but not for AppBuilder CLI. I think you can use the old logic and the new one. This way this code will work in both CLI. You just have to return `shelljs.error() === null` after the `catch`. This way if shelljs throws an error it will return `false` in the `catch` and if it doesn't throw but there is an error it will still return `false`.